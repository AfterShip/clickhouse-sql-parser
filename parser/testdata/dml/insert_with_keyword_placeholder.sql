INSERT INTO t (c) VALUES ({name :String});

