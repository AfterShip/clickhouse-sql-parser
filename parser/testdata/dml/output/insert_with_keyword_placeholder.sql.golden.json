[
  {
    "InsertPos": 0,
    "Format": null,
    "HasTableKeyword": false,
    "Table": {
      "Database": null,
      "Table": {
        "Name": "t",
        "QuoteType": 1,
        "NamePos": 12,
        "NameEnd": 13
      }
    },
    "ColumnNames": {
      "LeftParenPos": 14,
      "RightParenPos": 16,
      "ColumnNames": [
        {
          "Ident": {
            "Name": "c",
            "QuoteType": 1,
            "NamePos": 15,
            "NameEnd": 16
          },
          "DotIdent": null
        }
      ]
    },
    "Values": [
      {
        "LeftParenPos": 25,
        "RightParenPos": 40,
        "Values": [
          {
            "LeftBracePos": 26,
            "RightBracePos": 40,
            "Name": {
              "Name": "name",
              "QuoteType": 1,
              "NamePos": 27,
              "NameEnd": 31
            },
            "Type": {
              "Name": {
                "Name": "String",
                "QuoteType": 1,
                "NamePos": 33,
                "NameEnd": 39
              }
            }
          }
        ]
      }
    ],
    "SelectExpr": null
  }
]